xonsh/execer.py

```diff
@@ -68,7 +68,8 @@ def parse(self, input, ctx, mode='exec'):
         # tokens for all of the Python rules. The lazy way implemented here
         # is to parse a line a second time with a $() wrapper if it fails
         # the first time. This is a context-free phase.
-        tree, input = self._parse_ctx_free(input, mode=mode)
+        tree, input = self._parse_ctx_free(input, mode=mode,
+                                           wrap_subprocs=wrap_subprocs)
```

Comment:
So here is where I think that if `wrap_subprocs` is False, it should just 

```
return self.parser.parse(input, filename=self.filename, mode=mode, debug_level=(self.debug_level > 1))
```

Rather than adding arguments to this and the following function.